src/main/java/seedu/address/model/task/TaskName.java

```diff
@@ -7,7 +7,7 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidTaskName(String)}
  */
 public class TaskName {
-    public static final String MESSAGE_CONSTRAINTS = "Task name should not be empty or longer than 30 words.";
+    public static final String MESSAGE_CONSTRAINTS = "Task name should be no longer than 30 words.";
```

Comment:
Is there a reason why you modified this String? I thought it is fine to include the message constraint that a task name should not be empty.

Reply:
No, the correct one is the red one. I just realized think I have deleted the wrong line when resolving the conflicts 😆

Reply:
Okay, I have fixed it.